ui-manchette/src/components/Manchette.tsx

```diff
@@ -1,15 +1,18 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 
 import { ZoomIn, ZoomOut } from '@osrd-project/ui-icons';
 import cx from 'classnames';
 
 import { INITIAL_OP_LIST_HEIGHT, MAX_ZOOM_Y, MIN_ZOOM_Y } from './consts';
 import WaypointList from './WaypointList';
-import type { InteractiveWaypoint } from '../types';
+import type { InteractiveWaypoint, WaypointMenuData } from '../types';
+
+const WAYPOINT_MENU_LEFT_OFFSET = 8;
+const WAYPOINT_MENU_TOP_OFFSET = -2;
```

Comment:
Instead of having these consts, it would be nicer to move these into CSS. For instance:

```css
.menu-wrapper {
  left: 8px;
  margin-top: 30px;
}
```

Reply:
Done